### PR TITLE
Fix crm_simulate wrongly thinking stonith devices need restart

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@wenningerk , can you review this when you get a chance?

**Comment by kgaillot** [comment]
Updated and rebased on current master

**Comment by kgaillot** [comment]
Rebased on current master, no changes

**Comment by kgaillot** [comment]
rebased on master again to resolve a trivial conflict with the unit tests

File: lib/common/agents.c [hunk 1]
```diff
@@ -171,3 +171,42 @@ crm_provider_required(const char *standard)
...
+                         PCMK_STONITH_DELAY_MAX,
+                         PCMK_STONITH_HOST_ARGUMENT,
+                         PCMK_STONITH_HOST_CHECK,
+                         PCMK_STONITH_HOST_LIST,
+                         PCMK_STONITH_HOST_MAP,
+                         NULL)) {
+        return true;
+    }
+    param = strchr(param + 5, '_'); // Skip past "pcmk_ACTION"
+    return pcmk__str_any_of(param, "_action", "_timeout", "_retries", NULL);
```
Comment (wenningerk) [inline lib/common/agents.c:211]: probably safe enough but we could scan for pcmk_%s_%s

**Comment by wenningerk** [comment]
retest this please

**Comment by wenningerk** [comment]
lgtm

**Comment by kgaillot** [comment]
retest this please

**Pull request merged**